src/components/Panel/Panel.stories.tsx

```diff
@@ -0,0 +1,45 @@
+import type { Meta, StoryObj } from '@storybook/react'
+
+import { Panel } from './Panel'
+
+const meta: Meta<typeof Panel> = {
+  component: Panel
+}
+
+export default meta
+type Story = StoryObj<typeof Panel>
+
+const children =
+  'Integer et neque efficitur, lobortis nulla eget, porttitor augue. Integer eget tortor sed libero tincidunt semper vitae id mi.Integer et neque efficitur, lobortis nulla eget, porttitor augue. Integer eget tortor sed libero tincidunt semper vitae id mi.Integer et neque efficitur, lobortis nulla eget, porttitor augue. Integer eget tortor sed libero tincidunt semper vitae id mi.Integer et neque efficitur, lobortis nulla eget, porttitor augue. Integer eget tortor sed libero tincidunt semper vitae id mi.'
+
+export const PaddingDefault: Story = {
+  render: () => <Panel variant='padding-default'>{children}</Panel>
+}
```

Comment:
Albo usuniesz te story i zrobisz je na spokojnie w osobnym PR, albo trzeba je poprawic tak, żeby to wszystko dało się wyklikać w menu, a nie przechodzić przez kilka storiesów (tak jak jest w onebuilderze - https://storybookgha.s3.eu-west-1.amazonaws.com/index.html?path=/story/panel--light&args=padding:0)

Ja bym wybrała pierwszą opcję, żeby nie przeciągać tego Pr'a :)